microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -78,6 +78,15 @@ This section will continue to be updated over time to provide notes on enhanceme
 
 Issued: 2022-01-30
 
-{product-title} release 4.12.1 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:0452[RHBA-2023:0452] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0449[RHBA-2023:0449] advisory.
+{product-title} release 4.12.1 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:0452[RHBA-2023:0452] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0449[RHSA-2023:0449] advisory.
+
+For the TopoLVM refer to link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8]
```

Comment:
Also, not sure, but should it be `TopoLVM image`?

Reply:
@ShaunaDiaz @dhellmann could one of your confirm which is correct, please and thanks?

Reply:
@JoeAldinger Yes, it's an image.